versioned_docs/version-22.04/migrate/migrate-from-el-to-el.md

```diff
@@ -0,0 +1,209 @@
+
+   ```shell
+   rsync -avz /etc/centreon root@<IP_NEW_CENTREON>:/etc
+   rsync -avz /etc/centreon-broker root@<IP_NEW_CENTREON>:/etc
+   rsync -avz /var/log/centreon-engine/archives/ root@<IP_NEW_CENTREON>:/var/log/centreon-engine
+   rsync -avz --exclude centcore/ --exclude log/ /var/lib/centreon root@<IP_NEW_CENTREON>:/var/lib
+   ```
+
+   If you have given a custom name to your private key file, use the following format (just replace **id_rsa_custom** by your file name, and \<command> by the commands above):
+
+   ```shell
+   rsync -avz -e "ssh -i /root/.ssh/id_rsa_custom" <command>
+   ```
+
+   > Replace **<IP_NEW_CENTREON>** by the IP address of the new Centreon server.
+
+### Step 3: Retrieve the databases
+
+1. On the old server, create a dump of the databases:
+
+   ```shell
+   mysqldump -u root -p centreon > /tmp/centreon.sql
+   mysqldump -u root -p centreon_storage > /tmp/centreon_storage.sql
+   ```
+
+2. On the old server, stop MariaDB:
+
+   ```shell
+   service mysqld stop
+   ```
+
+3. On the old server, export the dumps to the new Centreon 22.04 database server (make sure you
+have enough space for large database dumps):
+
+   ```shell
+   rsync -avz /tmp/centreon.sql root@<IP_NEW_CENTREON>:/tmp/
+   rsync -avz /tmp/centreon_storage.sql root@<IP_NEW_CENTREON>:/tmp/
+   ```
+
+4. On the new database server, drop the original databases and
+create them again:
+
+   ```shell
+   mysql -u root -p
+   ```
+
+   ```SQL
+   DROP DATABASE centreon;
+   DROP DATABASE centreon_storage;
+   CREATE DATABASE centreon;
+   CREATE DATABASE centreon_storage;
+   exit
+   ```
+
+5. On the new database server, import the previously transfered dumps into the database:
+
+   ```shell
+   mysql -u root centreon -p </tmp/centreon.sql
+   mysql -u root centreon_storage -p </tmp/centreon_storage.sql
+   ```
+
+6. Upgrade the tables:
+
+   ```shell
+   mysql_upgrade
+   ```
+
+   If your database is password-protected, enter:
+
+   ```shell
+   mysql_upgrade -u <database_admin_user> -p
+   ```
+
+   Example: if your database_admin_user is `root`, enter:
+
+   ```shell
+   mysql_upgrade -u root -p
+   ```
+
+7. Start the mariadb process on the new server:
+
+   ```shell
+   systemctl start mariadb
+   ```
+
+> Replace **<IP_NEW_CENTREON>** by the IP address of the new Centreon server.
+
+### Step 4: Synchronize the plugins
+
+If you only use Centreon plugins, reinstall them on the new server:
+
+```shell
+yum install centreon-plugin-\*
+```
+
+If you are using custom plugins, synchronize the directories that contain your custom plugins, including any necessary dependencies.
+
+### Step 5: Upgrade Centreon
+
+1. On the new server, force the update by moving the contents of the
+**/var/lib/centreon/installs/install-22.04.0-YYYYMMDD\_HHMMSS** directory to
+the **/usr/share/centreon/www/install** directory:
+
+   ```shell
+   cd /var/lib/centreon/installs/
+   mv install-22.04.0-YYYYMMDD_HHMMSS/ /usr/share/centreon/www/install/
+   ```
```

Comment:
what version to use if older machine had an older 22.04 than the new one ?
idea: using the same version as the old one, to be sure to fully match the older machine conf and DB requests